controlplane/src/core/repositories/OrganizationRepository.ts

```diff
@@ -136,6 +145,7 @@ export class OrganizationRepository {
           userID: input.userId,
           organizationID: org.id,
         }),
+        limits: await this.getOrganizationLimits({ organizationID: org.id }),
```

Comment:
Can we fetch the limits in the organization query?

Reply:
you mean a join right?, yep we can

Reply:
Yes